app.js

```diff
@@ -116,66 +103,65 @@ function handleClicks(e) {
     removePhotos();
     alert('Thanks for your help, ' + userName + '! Click Ok to see your results');
+    refreshButton();
     updateClicks();
     updateShows();
-    showResults();
+    storeUserData();
     createChart();
+    Junk.totalClicks = 0;
   } else {
     randomJunk();
   }
 }
 
-// removes 3 images after 25 clicks
+//event listener to recognize validated clicks on images
+sectionEl.addEventListener('click', handleClicks);
+
+//hides 3 images after 25 clicks
 function removePhotos() {
-  leftEl.src = null;
+  leftEl.src = '';
   leftEl.alt = '';
-  centerEl.src = null;
+  centerEl.src = '';
   centerEl.alt = '';
-  rightEl.src = null;
+  rightEl.src = '';
   rightEl.alt = '';
 }
 
-//renders a table with survey results in addition to the chart
-function showResults() {
-  var h2El = document.getElementById("table-header");
-  h2El.textContent = userName + '\'s Results';
-
-  var trEl = document.createElement('tr');
-  var thEl = document.createElement('th');
-  thEl.textContent = 'Item';
-  trEl.appendChild(thEl);
-
-  for (var i in Junk.allJunk) {
-    thEl = document.createElement('th');
-    thEl.textContent = Junk.allJunk[i].name;
-    trEl.appendChild(thEl);
-  }
-
-  junkResults.appendChild(trEl);
+//switches skipButton to refreshButton
+function refreshButton() {
+  buttonEl.textContent = 'Let\'s do it again!';
+  buttonEl.style.backgroundColor = 'green';
+  buttonEl.addEventListener('click', clickRefreshButton);
+}
 
-  trEl = document.createElement('tr');
-  var tdEl = document.createElement('td');
-  tdEl.textContent = 'Views';
-  trEl.appendChild(tdEl);
-  for (i in Junk.allJunk) {
-    tdEl = document.createElement('td');
-    tdEl.textContent = Junk.allJunk[i].shows;
-    trEl.appendChild(tdEl);
-  }
+//refreshButton functionality
+function clickRefreshButton() {
+  buttonEl.removeEventListener('click', refreshButton);
+  randomJunk();
+  sectionEl.addEventListener('click', handleClicks);
+  skipButton();
+}
 
-  junkResults.appendChild(trEl);
+//creates skip button option
+function skipButton() {
+  buttonEl.addEventListener('click', clickSkipButton);
+  buttonEl.textContent = 'These suck. Skip \'em!';
+  buttonEl.style.backgroundColor = 'rgb(135, 161, 182);';
+}
 
-  trEl = document.createElement('tr');
-  tdEl = document.createElement('td');
-  tdEl.textContent = 'Clicks';
-  trEl.appendChild(tdEl);
-  for (i in Junk.allJunk) {
-    tdEl = document.createElement('td');
-    tdEl.textContent = Junk.allJunk[i].clicks;
-    trEl.appendChild(tdEl);
-  }
+//skipButton functionality
+function clickSkipButton() {
+  skipCounter++;
+  randomJunk();
+  console.log('SKIPPED! Skip Count =' + skipCounter + '. Item #s skipped: ' + Junk.lastShown);
+}
 
-  junkResults.appendChild(trEl);
+//stores user data AFTER survey complete
+function storeUserData() {
+  localStorage.userName = userName;
+  localStorage.itemNames = junkNames;
+  localStorage.chartClicks = junkClicks;
+  localStorage.chartViews = junkViews;
```

Comment:
Since `junkNames`, `junkClicks`, and `junkViews` are all arrays, you should store them in `localStorage` only after stringifying them. This will make retrieving the data much simpler.
Ex:
```javascript
localStorage.itemNames = JSON.stringify(junkNames);
```